src/java.base/share/classes/java/lang/StringUTF16.java

```diff
@@ -1531,6 +1531,7 @@ public static int lastIndexOfLatin1(byte[] src, int srcCount,
      * @return index of the most significant digit or minus sign, if present
      */
     static int getChars(int i, int index, byte[] buf) {
+        // Used by trusted callers.  Assumes all necessary bounds checks have been done by the caller.
```

Comment:
Now these are a bit redundant since these methods are surrounded by a comment to the same effect. I prefer inline comments like these, but we should avoid having both.

Reply:
Comments are redundant, but I can't find a solution